src/middleware/handle-request.ts

```diff
@@ -12,7 +12,7 @@ export async function handleRequest(
   { pathPrefix = "/api/github/oauth" }: HandlerOptions,
   request: OctokitRequest,
 ): Promise<OctokitResponse | undefined> {
-  if (request.method === "OPTIONS") {
+  if (pathname.startsWith(`${pathPrefix}/`) && request.method === "OPTIONS") {
```

Comment:
I think this line needs to be moved below 29 now, as `pathname` is not set until then

Reply:
Nice thanks! And actually seems like it may be better to just put that line 29 check above this one.. trying that now